### ISSUE TITLE
capacitive or inductive characteristics

### ISSUE THREAD

**Issue by tom-net** [yarda/chroma66202#4]
Hello, is it possible to read information from the Chroma 66202 meter via USB whether the reactive energy has capacitive or inductive characteristics?

**Comment by yarda** [comment]
I don't think so. If I am not missing anything it can measure reactive and apparent power, but IMHO it returns absolute values. I.e. it uses the following formula:
```
Reactive Power = sqrt((Vrms x Irms)^2 - (active_power)^2)
```

Active power is measured by sampling (integration).

To be 100% sure ask the vendor.

**Comment by tom-net** [comment]
Chroma softpanel displays sinusoidal current and voltage waveforms.

**Comment by yarda** [comment]
You can get raw data from it, either voltage or current, it's array of 2048 floats. Miminal window length is 0.1 s, which means maximal frequency of measurement cca. 10 kHz. Chroma 66202, FW 1.20. In this mode the device works like simple two channel ADC and you are on your own to visualize/process/analyze the raw data.
